scripts/mutate.py

```diff
@@ -72,6 +72,8 @@ class MutationType(Enum):
         "geq_ratio_ratio",
         "gt_ratio_ratio",
         "eq_ratio_ratio",
+        # most (all?) of these only appear on tests, but they are here for
+        # completeness' sake.
```

Comment:
Yeah, certainly all, since they are between `BEGIN_OCAML`/`END_OCAML`. In fact, most of `ratio.ml` is now in this state; at the end I might just integrate the parts of `ratio.ml` we need into `common.ml` and the parts that are test-only into test modules :sweat_smile: This is low priority though I guess.